CMakeLists.txt

```diff
@@ -654,8 +654,8 @@ endfunction()
 
 if(BUILD_TEST)
   set(TEST_BINARIES)
-  add_test(nvfuser_tests "${JIT_TEST_SRCS}" "")
-  list(APPEND TEST_BINARIES nvfuser_tests)
+  add_test(test_nvfuser "${JIT_TEST_SRCS}" "")
+  list(APPEND TEST_BINARIES test_nvfuser)
```

Comment:
Today, this is implemented as an exception in our CI filtering rule. Let's fix it, instead of making this an exception. It is annoying that after I have typed `test_` and pressed `tab` for completion, only to remember that nvfuser tests are not starting from `test_`.